src/26-ipfs-endeavors/index.md

```diff
@@ -0,0 +1,108 @@
+---
+# this is the final blog post's id (used in the directory)
+# this id is part of the url, and should only contain:
+#   letters, numbers, dashes.
+id: 26-ipfs-endeavors
+
+breadcrumbs:
+  - {name: "26-ipfs-endeavors", link: "./" }
+
+# the date here should be set to the final publication date,
+# on the day it is published.
+date: 2015-05-05
+
+# this is the Title
+title: IPFS Endeavors - An overview
+
+# this is the name of the main author(s)
+author: Juan Benet
+
+# technical details required for the software, don't change these.
+baseurl: ..
+template: tmpl/layouts/post.html
+collection: posts
+---
+
+In October, at Protocol Labs, Juan Benet sat down to do a quick overview of the main IPFS endeavors. This video runs through the different projects; a transcript is provided below.
```

Comment:
minor comment: this was part of IPFS calls, not protocol labs. the "at protocol labs" "making it a place" thing is just also misleading in isolation here.